pages/agent/v3/cli_meta_data.md.erb

```diff
@@ -14,6 +14,8 @@ Use this command in your build scripts to save string data in the Buildkite meta
 <%= render 'agent/v3/help/meta_data_set.txt' %>
 ```
 
+Meta-data values are restricted to a maximum of 100 kilobytes. Keys and values larger than 1 kilobyte are deprecated. Please use [artifacts](https://buildkite.com/docs/agent/v3/cli-artifact) for large data which needs to uploaded and downloaded.
```

Comment:
Are we planning to make the deprecation the standard at some point? If so, do we have an idea when?

Reply:
We haven't decided on an official deprecation yet, so I've changed the wording to "discouraged".